graaljs/graaljs-maven-webpack-qrcode/README.md

```diff
@@ -0,0 +1,330 @@
+# Using Node Packages in a Java Application
```

Comment:
I would use a more specific name: "GraalJS QRCode Demo".
To be in sync with GraalPy demos recently added by @fniephaus , for example "GraalPy JBang QRCode Demo"
https://github.com/graalvm/graalvm-demos/pull/291/files#diff-381004f1f683b64bf7c635fc4632174d8696ee172ab55486d87836f9255e2dc4R1

Reply:
I think we can make this more consistent later on. I like Andrea's suggestion, because it makes clear what to learn from the demo, not what specific dependencies are used.